src/hotspot/share/opto/vectornode.hpp

```diff
@@ -974,6 +976,7 @@ class StoreVectorNode : public StoreNode {
    virtual uint match_edge(uint idx) const { return idx == MemNode::Address ||
                                                     idx == MemNode::ValueIn ||
                                                     idx == MemNode::ValueIn + 1; }
+   virtual Node* offsets() const { return in(Offsets); }
```

Comment:
Would be nice to add some `override` keywords here ;)

Reply:
And also below.

Reply:
On macosx we build with the flag `-Winconsistent-missing-override` enabled, which seems to allow either no `override` anywhere or in all places where a method actually overrides another one. So, in this case it seems that we would either need to add `override`s to `virtual int Opcode()`, `virtual uint match_edge(uint idx)`, `virtual Node* Ideal(PhaseGVN* phase, bool can_reshape)` etc. in all classes, or remove the 4 `override` keywords from the `indices` and `mask` methods. I might opt for the second choice. @eme64 what do you think?

Reply:
The second choice makes sense to limit the scope of the changes in this PR. If we still want to add the `override` keywords, we should do that in a separate RFE.

Reply:
Ah yes. I forgot about `-Winconsistent-missing-override`. No `override` keyword then :disappointed:

Reply:
C'est la vie 😉 
Removed.
Thanks guys.